Content.Client/UserInterface/Systems/Chat/Controls/ChannelFilterPopup.xaml.cs

```diff
@@ -86,7 +87,7 @@ public void SetChannels(ChatChannel channels)
 
     private void CheckboxPressed(ButtonEventArgs args)
     {
-        var checkbox = (ChannelFilterCheckbox) args.Button;
+        var checkbox = (ChannelFilterCheckbox)args.Button;
```

Comment:
untouch

Reply:
urgh istg I removed all of those :pensive: